source/commands/danger-pr.ts

```diff
@@ -1,5 +1,50 @@
 import * as program from "commander"
+import * as debug from "debug"
+import * as fs from "fs"
+import { FakeCI } from "../ci_source/providers/Fake"
+import { GitHub } from "../platforms/GitHub"
+import { Executor } from "../runner/Executor"
+import { pullRequestParser } from "../platforms/github/pullRequestParser"
+import { runDangerfileEnvironment } from "../runner/DangerfileRunner"
 
-program.parse(process.argv)
+const d = debug("danger:pr")
 
-console.log("Not Yet implmented")
+program
+  .option("-d, --dangerfile [filePath]", "Specify custom dangefile other than default dangerfile.js")
+  .parse(process.argv)
+
+const dangerFile = (program as any).dangerfile || "dangerfile.js"
+const stat = fs.statSync(dangerFile)
```

Comment:
`fs.statSync` will throw exception if file does not exists, so should catch for non-existing file cases.